client/src/styles/Header/Header.scss

```diff
@@ -0,0 +1,163 @@
+nav {
+  margin: 0;
+  overflow: hidden;
+  background: black;
+  height: 50px;
+}
+
+.nav-content {
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 0 15px;
+  display: flex;
+  justify-content: space-between;
+  margin-top: 2px;
+
+  a {
+    display: block;
+    font-size: 14px;
+    line-height: 44px;
+    text-decoration: none;
+    transition: all 0.3s;
+    color: #ffffff;
+  }
+}
+
+.icon {
+  opacity: 0.8;
+  margin-top: 8px;
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.nav-links {
+  width: 75%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  a {
+    opacity: 0.8;
+
+    &:hover {
+      opacity: 1;
+    }
+  }
+}
+
+.hra {
+  display: none;
+}
```

Comment:
You give `display: none` to `.hra` here... But nowhere it is shown with the right display. So what's the use? Where are you displaying it at all?

Reply:
I was actually trying to implement theese horizontal lines, but it wasn't working, so I removed it from the .js file, but forgot to remove it from here😅 will do that